src/glassflow/pipelines.py

```diff
@@ -240,3 +232,24 @@ def consume_failed(self) -> operations.ConsumeFailedResponse:
                                      http_res)
 
         return res
+
+    def _get_headers(
+            self, request: dataclasses.dataclass,
+            req_content_type: Optional[str] = None
+    ) -> dict:
+
+        headers = utils.get_req_specific_headers(request)
+        headers['Accept'] = 'application/json'
+        headers['Gf-Client'] = (
+            f'{self.glassflow_client.glassflow_config.user_agent} / '
+            f'{self.glassflow_client.glassflow_config.sdk_version}')
+        headers['user-agent'] = headers['Gf-Client']
```

Comment:
If I understand correctly, with these changes it sends:
```
Gf-Client: glassflow-python-sdk/0.0.2
User-Agent: glassflow-python-sdk/0.0.2
```

Does it make sense to send it like that?
```
Gf-Client: python-sdk/0.0.2
User-Agent: glassflow-python-sdk/0.0.2
```

Reply:
It makes sense, however I wonder if it will be extra code for the sake of it.

For analytics, we only need one of them, so effectively, the `User-Agent` will likely not be used (I would imagine the logic on the DWH being, use `User-Agent` when `Gf-Client` is not present).

Reply:
I like the idea of having the headers different and it's not so much code